Managers/PerformanceMan.cpp

```diff
@@ -183,45 +183,46 @@ namespace RTE {
 	void PerformanceMan::DrawPeformanceGraphs(AllegroBitmap &bitmapToDrawTo) {
 		CalculateSamplePercentages();
 
+		GUIFont *guiFont = g_FrameMan.GetLargeFont();
 		char str[128];
 
 		for (int pc = 0; pc < PerformanceCounters::PerfCounterCount; ++pc) {
 			int blockStart = c_GraphsStartOffsetY + pc * c_GraphBlockHeight;
 
-			g_FrameMan.GetLargeFont()->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX, blockStart, m_PerfCounterNames[pc], GUIFont::Left);
+			guiFont->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX, blockStart, m_PerfCounterNames[pc], GUIFont::Left);
 
-			// Print percentage from PerformanceCounters::SimTotal
 			int perc = static_cast<int>((static_cast<float>(GetPerformanceCounterAverage(static_cast<PerformanceCounters>(pc))) / static_cast<float>(GetPerformanceCounterAverage(PerformanceCounters::SimTotal)) * 100));
 			std::snprintf(str, sizeof(str), "%%: %u", perc);
-			g_FrameMan.GetLargeFont()->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX + 60, blockStart, str, GUIFont::Left);
+			guiFont->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX + 60, blockStart, str, GUIFont::Left);
 
-			// Print average processing time in ms
+			// Print average processing time in milliseconds.
 			std::snprintf(str, sizeof(str), "T: %llu", GetPerformanceCounterAverage(static_cast<PerformanceCounters>(pc)) / 1000);
-			g_FrameMan.GetLargeFont()->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX + 96, blockStart, str, GUIFont::Left);
+			guiFont->DrawAligned(&bitmapToDrawTo, c_StatsOffsetX + 96, blockStart, str, GUIFont::Left);
 
 			int graphStart = blockStart + c_GraphsOffsetX;
 
-			// Draw graph backgrounds
+			// Draw graph backgrounds.
 			bitmapToDrawTo.DrawRectangle(c_StatsOffsetX, graphStart, c_MaxSamples, c_GraphHeight, 240, true);
 			bitmapToDrawTo.DrawLine(c_StatsOffsetX, graphStart + c_GraphHeight / 2, c_StatsOffsetX - 1 + c_MaxSamples, graphStart + c_GraphHeight / 2, 96);
 
-			// Draw sample dots
+			// Draw sample dots.
 			int peak = 0;
 			int sample = m_Sample;
 			for (int i = 0; i < c_MaxSamples; ++i) {
-				// Show microseconds in graphs, assume that 33333 microseconds (one frame of 30 fps) is the highest value on the graph
-				int value = std::clamp(static_cast<int>(static_cast<float>(m_PerfData[pc].at(sample)) / (1000000.0F / 30.0F) * 100.0F), 0, 100);
+				// Show microseconds in graphs, assume that the RealToSimCap (33333 microseconds (one frame of 30 fps) by default) is the highest value on the graph. The graph will scale with the RealToSimCap if it is changed.
```

Comment:
lol did you just write stuff in brackets inside stuff in brackets in this comment? That's definitely illegal!